lib/handler/status.c

```diff
@@ -112,7 +112,7 @@ static void send_response(struct st_h2o_status_collector_t *collector)
     resp[cur_resp++] = (h2o_iovec_t){H2O_STRLIT("\n}\n")};
 
     req->res.status = 200;
-    h2o_add_header(&req->pool, &req->res.headers, H2O_TOKEN_CONTENT_TYPE, H2O_STRLIT("text/plain; charset=utf-8"));
+    h2o_add_header(&req->pool, &req->res.headers, H2O_TOKEN_CONTENT_TYPE, H2O_STRLIT("application/json; charset=utf-8"));
```

Comment:
Is this change relevant to the PR?
(and I believe we should use `text/plain` considering the fact that it can be displayed using web browsers...)